src/porepy/grids/partition.py

```diff
@@ -45,7 +45,10 @@ def partition_metis(g: pp.Grid, num_part: int) -> np.ndarray:
     c2c = g.cell_connection_map()
 
     # Convert the cells into the format required by pymetis
-    adjacency_list = [list(c2c.getrow(i).indices) for i in range(c2c.shape[0])]
+    indptr = c2c.indptr
```

Comment:
Would it be wise to do a type check/conversion here to ensure no problems with csr vs csc?